content/docs/04-clusters/03-edge/09-site-deployment/01-model-profile.md

```diff
@@ -87,6 +87,101 @@ Consider creating additional profiles with out-of-the-box packs for monitoring,
+
+
+Use the following steps to deploy Edge clusters using your cluster profile with the BYOOS Edge OS.
+
+# Prerequisites
+
+- Provider images are built.
+
+
+- ` system.uri`should be constrcuted. 
+
+# Enablement
+
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a **Tenant Admin**.
+
+
+2. Navigate to the left **Main Menu** and select **Profiles**.
+
+
+3. Click on **Add Cluster Profile**. 
+
+
+4. Provide **Basic Information**, such as profile name and description. Select **Full** and click on **Next**. <br /> <br />
+
+    <InfoBox>
+
+    Refer to the [Creating Cluster Profiles](/https://docs.spectrocloud.com/cluster-profiles/task-define-profile) resource to learn about creating a new cluster profile.
+
+    </InfoBox>
+
+5. Select **Edge Native** as the **Cloud Type** and click on **Next**.
+
+
+6. Select **Public Repo** in the **Registry field**.
+
+
+7. Select **BYOOS Edge OS** in the **Pack Name** field and the pack version. 
+
+
+8. The pack editor displays on the right section of the dashboard where you can customise the BYOOS pack. 
+
+## Installer Parameters
+
+| Parameter            | Description                                            |
+|----------------------|--------------------------------------------------------|
+| `pack:content:` | Specifies the content of the BYOOS pack. |
+| `pack.content.images` | Specifies the list of images to use for the OS pack. |
+| `pack.content.images.-  images` | Specifies an image to use for the OS pack. |
+
+
+## User Data Parameters
+
+| Parameter            | Description                                            |
+|----------------------|--------------------------------------------------------|
+| `options.system.uri:` | The system URI specifying the location of the BYOOS image. |
+| `image.registry` | The domain of the registry where the BYOOS image is stored. |
+| `image.repo` | The name of the BYOOS image repository. |
+| `image.palette.edge.version` | The Palette Edge software version used in the BYOOS container image. |
+| `image.client.tag` |  The tag given to the image used for the BYOOS, specifying its version. |
+
+
+Example BYOOS Pack Customization File
+
+<br />
+
+```yaml
+pack:
+ content:
+   images: 
+    - image: ‘{{.spectro.pack.edge-native-byoi.options.system.uri}}’
+    
+options: 
+ system.uri: “{{.spectro.pack.edge-native-byoi.image.registry}}/{{.spectro.pack.edge-native-byoi.image.repo}}:{{.spectro.system.kunernetes.version}}-{{.spectro.pack.edge-native-byoi.image.version}}_{{.spectro.pack.edge-native-byoi.image.client.tag}}”
+
+image: 
+ registry: "" 
+ repo: ""
+ palette.edge.version: ""
+ client.tag: ""
+```
+
+<br />
+
+Refer to the  [BYOOS Pack](/) resource to learn more about the pack details.
```

Comment:
Missing doc URL. 
Also, how are we referring to the pack? Are we just calling it the BYOOS pack because there's one for Edge and one for non-Edge?